portfolio.html

```diff
@@ -176,5 +176,85 @@ <h1>About me</h1>
         </ul>
       </div>
     </section>
+    <section id="contact">
+      <div>
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <div>
+        <form action="https://formspree.io/f/mnqwzykj" method="post">
+          <ul>
+            <li>
+              <label for="fullname"></label>
+              <input
+                type="text"
+                id="fullname"
+                name="user_name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <label for="email"></label>
+              <input
+                type="email"
+                id="email"
+                name="user_email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <label for="msg"></label>
+              <textarea
+                name="user_message"
+                id="msg"
+                cols="30"
+                rows="10"
+                placeholder="Enter text here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li id="footer-button">
+              <button type="submit">Get in touch</button>
+            </li>
+          </ul>
+        </form>
+      </div>
+      <div>
+        <hr class="footer-solid" />
+      </div>
+      <div id="footersocial">
+        <img
+          class="social"
+          src="/images/Icons/Social/footericons/cat.png"
+          alt="cat icon"
+        />
```

Comment:
OPTIONAL
Consider fixing the relative path for your images cause they may not be visible in all possible scenarios such as my case. Use `"./images/Icons/Social/footericons/cat.png"` instead of what you are using.

On changing the relative path, here is what I have on display.
![Screenshot from 2022-03-22 23-40-36](https://user-images.githubusercontent.com/65501535/159572031-f1d0fc6a-a476-4f8f-b1b0-268400abcb86.png)